index.html

```diff
@@ -313,6 +313,29 @@ <h3 class="text-center">Newsletter <small>Subscribe to get product updates about
     </div>
 </div>
 
+<a name="get-started"></a>
+<div class="container-fluid">
+    <div class="container text-centeR">
+
+    <hr>
+        <div class="row" id="get-started">
+            <div id="hosted" class="col-md-6">
+                <h2><img src="static/img/logo_white.png"> <label>HOSTED</label></h2>
+                <p>Start querying and sharing in a few clicks with our hosted service.</p>
+                <strong>Instant and hassle free</strong>
+                <a href="https://github.com/getredash/redash">Early access signup</a>
+            </div>
+            <span>or</span>
+            <div id="opensource" class="col-md-6">
+                <h2><img src="static/img/logo_white.png"> <label>0.9.1</label></h2>
+                <p>Download the free and open source version on your own server.</p>
```

Comment:
Maybe: 
"Setup Re:dash on-premise with our cloud images or setup script."
OR:
"Setup the free and open source Re:dash on-premise with a cloud image, Docker image or setup script."

Not sure about "free and open source", as it might signal that the hosted version isn't open source (which it is).